buildbot/buildbot/worker.py

```diff
@@ -154,6 +154,8 @@ def _task_config(task, config):
     task['sdsflags'] = flags
     task['estimate'] = est
 
+    task['directives'] = task['config'].get('directives') or \
+        config['DEFAULT_DIRECTIVES']
```

Comment:
Accordingly with the above, I think it's fine for this to just leave off the `or` part and instead pass `''` as the second parameter to `.get()`, which will make this an empty string by default.